### ISSUE TITLE
Error when install in macOS Mojave

### ISSUE THREAD

**Issue by LydByDissing** [rancher-sandbox/rancher-desktop#920]
When going through the install wizard, the following error occurs: `Error Starting Kubernetes`
Detailed message:
`There was an unknown error starting Kubernetes: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1`

**To Reproduce**
Steps to reproduce the behaviour:
1. Download release 0.6.1 for macOS 
2. Open Rancher desktop app
3. select Kubernetes version 1.21.6.
4. toggled RAM setting to 2GB

**Result**
![Skærmbillede 2021-11-07 kl  10 03 56](https://user-images.githubusercontent.com/63507762/140638878-e6970576-1e7d-4e3c-927f-0fe57173a8c0.png)

**Expected behaviour**
A running k3s install etc.

**Setup (please complete the following information):**
 - macOS Mojave (10.14.6)
 - Rancher Desktop version 0.6.1
 - Kubernetes version 1.21.7

**Comment by kkzz8888** [comment]
same issue here with macOS Big Sur (11.16.1), version 0.6.1 (0.6.0 did work) for Rancher Desktop and k8s version 1.21.6.

**Comment by ericpromislow** [comment]
Hi, next time this happens could you zip up the pertinent logs in `~/Library/Logs/rancher-desktop` (most likely `lima*.log`, `k3s.log`, and `k8s.log`) (checking for sensitive strings of course) and get them to us. While you could attach it here, probably better to contact one of us and send it directly)

**Comment by mikefut** [comment]
Hi. I had the same problem, reading through the logs revealed the following error:
`background.log:Kubernetes was unable to start: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/bin/kubectl exited with code 1
k8s.log:error: open /Users/.../.kube/config.lock: file exists
`
Removing a .lock file fixed the issue. However, I'm not sure what else may break by removing it :))

**Comment by LydByDissing** [comment]
> same issue here with macOS Big Sur (11.16.1), version 0.6.1 (0.6.0 did work) for Rancher Desktop and k8s version 1.21.6.

Will try to downgrade and report back

**Comment by LydByDissing** [comment]
> Hi. I had the same problem, reading through the logs revealed the following error:
> `background.log:Kubernetes was unable to start: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/bin/kubectl exited with code 1 k8s.log:error: open /Users/.../.kube/config.lock: file exists `
> Removing a .lock file fixed the issue. However, I'm not sure what else may break by removing it :))

No such .lock file. It seems docker desktop created a ~/.kube/config; which makes sense. Deleting this, did nothing.

**Comment by pboushy** [comment]
Same happens in 10.14.6 with Kubernetes v1.19.15

Logs:
```time="2021-12-09T13:55:50-07:00" level=info msg="Terminal is not available, proceeding without opening an editor"
time="2021-12-09T13:55:50-07:00" level=fatal msg="failed to run [system_profiler SPNetworkDataType -json]: stdout=\"\", stderr=\"Usage: system_profiler [-listDataTypes]\\n       system_profiler [-xml] [-timeout n] [-detailLevel n]\\n       system_profiler [-xml] [-timeout n] [dataType1 ... dataTypeN]\\n\\n  -detailLevel n    specifies the level of detail for the report\\n                      mini = short report (contains no identifying or personal information)\\n                      basic = basic hardware and network information\\n                      full = all available information\\n\\n  -listDataTypes    lists all the available datatypes\\n\\n  -xml              generates xml output instead of plain text\\n                    if redirected to a file with the extension \\\".spx\\\"\\n                    the file can be opened in System Profiler.app\\n\\n  -timeout          specifies the maximum time to spend gathering information\\n                    the default is 180 seconds, 0 means no timeout\\n\\n  Redirect stderr to /dev/null to suppress progress and error messages.\\n\\nExamples:\\n\\n  system_profiler\\n     Generates a text report with the standard detail level.\\n\\n  system_profiler -detailLevel mini\\n     Generates a short report without identifying/personal information.\\n\\n  system_profiler -listDataTypes\\n     Shows a list of the available data types.\\n\\n  system_profiler SPSoftwareDataType SPNetworkDataType\\n     Generates a text report containing only software and network data.\\n\\n  system_profiler -xml >MySystem.spx\\n     Creates a XML file which can be opened by System Profiler.app\\n\\n\": exit status 1"
+ limactl start --tty=false /Users/pboushy/Library/Application Support/rancher-desktop/lima/_config/0.yaml
Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/Applications/Rancher Desktop.app/Contents/Resources/app.asar/dist/app/background.js:1:7901)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
Error starting lima: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/Applications/Rancher Desktop.app/Contents/Resources/app.asar/dist/app/background.js:1:7901)
    at ChildProcess.emit (events.js:315:20)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
```

Error is that `limactl` is running system_profiler incorrectly. `-json` argument is invalid in 10.14.

You have to use `-xml` and parse a plist (xml) instead.

**Comment by pboushy** [comment]
FYI - it looks like the issue is resolved in lima v0.7.2 (released Oct/22/2021) based on this https://github.com/lima-vm/lima/pull/353

It looks like the current version of Rancher Desktop include limactl v0.7.0 (released Oct/08/2021):
```'/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl' --version
limactl version e494f74```

https://github.com/lima-vm/lima/commit/e494f7450b6100149bf6adefa9becfbfacbafdad


Current version of limactl is v0.7.4 (released Nov/22/2021)

**Comment by jandubois** [comment]
This should be fixed once #1085 is merged, if it isn't already.
